project-fob/Models/Fob.cs

```diff
@@ -1,37 +1,69 @@
-﻿using project_fob.Data;
+﻿using Microsoft.EntityFrameworkCore;
+using project_fob.Data;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 
 
-namespace project_fob.Models 
+namespace project_fob.Models
 {
-    public class Fob 
+    public class Fob
     {
         [Key]
         public int Id { get; set; }
-        public int AttendeeCount { get; set; }
-        public int FobCount { get; set; }
+        
+        //This variable and code is mainly here because of past values are bein loaded into the project.
+        public int AttendeeCount
+        {
+            get
+            {
+                if (Meeting != null)
+                { return -1; }
+                else { return -11; }
+            }
+            set { }
+        }
+        public int FobCount { get { return Fobbed.Count; } set { } }
```

Comment:
Having an empty setter on `AttendeeCount` and `FobCount` are probably going to cause problems when they classes get mapped to & from database tables. If you don't want Entity Framework to try mapping these to the database, you can add the `[NotMapped]` attribute to the properties. There's more over in the [EF Core docs](https://docs.microsoft.com/en-us/ef/core/modeling/included-properties)